statsnewsletter.php

```diff
@@ -42,6 +46,16 @@ public function __construct()
 		$this->author = 'PrestaShop';
 		$this->need_instance = 0;
 
+		if (version_compare(_PS_VERSION_, '1.7.0.0', '>=')) {
+			$this->table_name = _DB_PREFIX_ . 'emailsubscription';
+			$this->newsletter_module_name = 'ps_emailsubscription';
+			$this->newsletter_module_human_readable_name = 'Email subscription';
```

Comment:
The complete sentence is translated [here](https://github.com/PrestaShop/statsnewsletter/pull/1/commits/5947dffaca466ef36547a15bea2312e1064aa033#diff-c09969987f5e785ca9a6bf24d64ce6f6R102)